src/chainparams.cpp

```diff
@@ -164,7 +164,10 @@ class CMainParams : public CChainParams {
         };
 
         m_assumeutxo_data = MapAssumeutxo{
-         // TODO to be specified in a future patch.
+            {
+                685000,
```

Comment:
Optimistically bumping the block, but also making sure I don't keep putting the wrong numbers here:

```cpp
/*height=*/780000,
{AssumeutxoHash{uint256S(/*txoutset_hash=*/"0x97d2989ab4c1b2cfb4eaa37788aeea08a3a6757eabd94ae9b7cdde07e79381da")},
/*nchaintx=*/812391117},
```

Maybe also add a string initialiser to `AssumeutxoHash` to avoid the extra `uint256S`.